src/stat.cc

```diff
@@ -12,6 +12,9 @@
 #include "AccessLogEntry.h"
 #include "CacheDigest.h"
 #include "CachePeer.h"
+#if USE_CACHE_DIGESTS
+#include "CachePeers.h"
+#endif
```

Comment:
Your call, but I would not imply that we should start creating complex conditions for our own header inclusion (where those conditions are not necessary to avoid some serious problems).

```suggestion
#include "CachePeers.h"
```